### PR TITLE
Initial Documentation

### PR DESCRIPTION
**Pull request by matthewrsj**
* Add usage documentation
* Update method docstrings
* Update README

### CODE REVIEW

**Comment by matthewrsj** [comment]
Fixes #22

**Comment by matthewrsj** [comment]
Documentation included for #23

File: source/usage.rst [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+or a list of times from TimeSync (**get_times()**), and get a project or list of
+projects from TimeSync (**get_projects()**).
+
+Pymesync currently supports the following TimeSync API versions:
+
+* v1
+
+All of these methods return a python list of one to many dictionaries.
+
+* **send_time(baseurl)** - Sends time to baseurl (TimeSync), must include version
```
Comment (Kennric) [inline source/usage.rst]: This doesn't look right - send time just takes a time, and baseurl is passed to the constructor, right?

Comment (matthewrsj) [inline source/usage.rst]: shoot, you're right

**Comment by Kennric** [comment]
We generally put docs in a docs/ subdirectory, and don't include the built docs int he repo, just the source. When setting up Sphinx, you can choose separate source and build dirs, we usually do this and then git ignore docs/build - docs can be put on Read the Docs, or built by the user with their own sphinx installation.

**Comment by matthewrsj** [comment]
Ok, will fix

**Comment by Kennric** [comment]
+1

**Pull request merged**